content/developer/Theme/Icons/SuitePIcons/Sidebar_icons.adoc

```diff
@@ -0,0 +1,11 @@
+---
+title: "Sidebar Icons  "
+weight: 20
+pre: "<b> </b>"
+---
+
+:imagesdir: /images/en/developer/theme-icons/
+
+'''
+
+image:Sidebar_image.png[title="Sidebar image"]
```

Comment:
This image is very large, can it be made smaller?

Reply:
resolution or file size? @pedro-gr

Reply:
The idea is to reduce file size, but the way to get there I am not so sure, you'll have to try a few things and see what gives nice results. I would try resampling to a smaller resolution if that doesn't make it too coarse...